plugins/modules/system/capabilities.py

```diff
@@ -110,10 +110,16 @@ def getcap(self, path):
         #   '/foo'
         # If the file does not exist the output will be (with rc == 0...):
         #   '/foo (No such file or directory)'
```

Comment:
Is this case still relevant?

Reply:
```
[mator@archlinux ~]$ getcap /foo
/foo (No such file or directory)
[mator@archlinux ~]$ echo $?
0
[mator@archlinux ~]$ getcap /foo 2>/dev/null
[mator@archlinux ~]$ echo $?
0
```
still valid, yes

Reply:
Doesn't that mean that your change isn't working correctly, since now it will do `caps = stdout.split()[1].strip().split()` instead of calling `module.fail_json`?

Reply:
@felixfontein if you mean that the output (stdout) will be error message, you're wrong, since stderr would have a message, stdout will be empty (per my comment) and RC still would be 0